[zh] sync docs/reference/config-api/apiserver-config.v1beta1.md (#41172)
kubernetes/website: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md

---
title: kube-apiserver 配置 (v1beta1)
content_type: tool-reference
package: apiserver.k8s.io/v1beta1
---
<!-- 
title: kube-apiserver Configuration (v1beta1)
content_type: tool-reference
package: apiserver.k8s.io/v1beta1
auto_generated: true
-->

<!-- 
Package v1beta1 is the v1beta1 version of the API.</p> 
-->

> <p>包 v1beta1 包含 API 的 v1beta1 版本。</p>

Reviewer: ```suggestion
<p>v1beta1 包是 v1beta1 版本的 API。</p>
```

<tr><td><code>kind</code><br/>string</td><td><code>TracingConfiguration</code></td></tr>

<tr><td><code>TracingConfiguration</code> <B><!--[Required]-->[必需]</B><br/>
<a href="#TracingConfiguration"><code>TracingConfiguration</code></a>
</td>
<td>
<!-- (Members of <code>TracingConfiguration</code> are embedded into this type.) -->
（<code>TracingConfiguration</code> 的成员嵌入到这种类型中。）
   <p>
   <!-- 
   Embed the component config tracing configuration struct 
   -->
   嵌入组件配置中的跟踪配置结构体。
   </p>
</td>
</tr>
</tbody>
</table>

## `Connection`     {#apiserver-k8s-io-v1beta1-Connection}

<!--
**Appears in:**
-->
**出现在：**

- [EgressSelection](#apiserver-k8s-io-v1beta1-EgressSelection)

<p>
<!--
Connection provides the configuration for a single egress selection client.
-->
Connection 提供某个 Egress 选择客户端的配置信息。
</p>

<table class="table">
<thead><tr><th width="30%"><!-- Field-->字段</th><th><!--Description-->描述</th></tr></thead>
<tbody>

<tr><td><code>proxyProtocol</code> <B><!--[Required]-->[必需]</B><br/>
<a href="#apiserver-k8s-io-v1beta1-ProtocolType"><code>ProtocolType</code></a>
</td>
<td>
   <p>
   <!-- 
   Protocol is the protocol used to connect from client to the konnectivity server. 
   -->

>    Protocol 是客户端连接到 konnectivity 服务器所使用的协议。

Reviewer 2: ```suggestion
   proxyProtocol 是客户端连接到 konnectivity 服务器所使用的协议。
```

   </p>
</td>
</tr>
<tr><td><code>transport</code><br/>
<a href="#apiserver-k8s-io-v1beta1-Transport"><code>Transport</code></a>
</td>
<td>
   <p>
   <!-- 
   Transport defines the transport configurations we use to dial to the konnectivity server.
   This is required if ProxyProtocol is HTTPConnect or GRPC. 
   -->
   transport 定义的是传输层的配置。我们使用这个配置来联系 konnectivity 服务器。
   当 proxyProtocol 是 HTTPConnect 或 GRPC 时需要设置此字段。
   </p>
</td>
</tr>
</tbody>
</table>

## `EgressSelection`     {#apiserver-k8s-io-v1beta1-EgressSelection}

<!--
**Appears in:**
-->
**出现在：**

- [EgressSelectorConfiguration](#apiserver-k8s-io-v1beta1-EgressSelectorConfiguration)

<p>
<!--
EgressSelection provides the configuration for a single egress selection client.
-->
EgressSelection 为某个 Egress 选择客户端提供配置信息。
</p>

<table class="table">
<thead><tr><th width="30%"><!--Field-->字段</th><th><!--Description-->描述</th></tr></thead>
<tbody>
  
<tr><td><code>name</code> <B><!--[Required]-->[必需]</B><br/>
<code>string</code>
</td>
<td>
   <p>

>    <!-- name is the name of the egress selection.
>    Currently supported values are &quot;controlplane&quot;, &quot;master&quot;, &quot;etcd&quot; and &quot;cluster&quot;
>    The &quot;master&quot; egress selector is deprecated in favor of &quot;controlplane&quot; -->
>   name 是 Egress 选择器的名称。当前支持的取值有 &quot;controlplane&quot;，
>   &quot;master&quot;，&quot;etcd&quot; 和 &quot;cluster&quot;。
>   &quot;master&quot; Egress 选择器已被弃用，推荐使用 &quot;controlplane&quot;。

Reviewer: ```suggestion
   <!--
   name is the name of the egress selection.
   Currently supported values are &quot;controlplane&quot;, &quot;master&quot;, &quot;etcd&quot; and &quot;cluster&quot;
   The &quot;master&quot; egress selector is deprecated in favor of &quot;controlplane&quot;
   -->
   name 是 Egress 选择算符的名称。当前支持的取值有 &quot;controlplane&quot;、
   &quot;master&quot;、&quot;etcd&quot; 和 &quot;cluster&quot;。
   &quot;master&quot; Egress 选择算符已被弃用，推荐使用 &quot;controlplane&quot;。
```
参考 [en-zh glossary](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/)，一个页面上可以把`选择器`和`选择算符`统一一下。

   </p>
</td>
</tr>
<tr><td><code>connection</code> <B><!--[Required]-->[必需]</B><br/>
<a href="#apiserver-k8s-io-v1beta1-Connection"><code>Connection</code></a>
</td>
<td>
   <p>
   <!--
   connection is the exact information used to configure the egress selection
   -->

>    connection 是用来配置 Egress 选择器的配置信息。

Reviewer 2: ```suggestion
   connection 是用来配置 Egress 选择算符的配置信息。
```

   </p>
</td>
</tr>
<tr><td><code>tlsConfig</code><br/>
<a href="#apiserver-k8s-io-v1beta1-TLSConfig"><code>TLSConfig</code></a>
</td>
<td>
   <p>
   <!--
   TLSConfig is the config needed to use TLS when connecting to konnectivity server
   -->
   tlsConfig 是使用 TLS 来连接 konnectivity 服务器时需要的信息。
   </p>
</td>
</tr>
</tbody>
</table>

## `TLSConfig`     {#apiserver-k8s-io-v1beta1-TLSConfig}

<!--
**Appears in:**
-->
**出现在：**

- [TCPTransport](#apiserver-k8s-io-v1beta1-TCPTransport)

<p>
<!--
TLSConfig provides the authentication information to connect to konnectivity server
Only used with TCPTransport
-->
TLSConfig 为连接 konnectivity 服务器提供身份认证信息。仅用于 TCPTransport。
</p>

<table class="table">
<thead><tr><th width="30%"><!--Field-->字段</th><th><!--Description-->描述</th></tr></thead>
<tbody>

<tr><td><code>caBundle</code><br/>
<code>string</code>
</td>
<td>
   <p>
   <!--
   caBundle is the file location of the CA to be used to determine trust with the konnectivity server.
   Must be absent/empty if TCPTransport.URL is prefixed with http://
   If absent while TCPTransport.URL is prefixed with https://, default to system trust roots.
   -->

>    caBundle 是指向用来确定与 konnectivity 服务器间信任欢喜的 CA 证书包的文件位置。

Reviewer: ```suggestion
   caBundle 是指向用来确定与 konnectivity 服务器间信任关系的 CA 证书包的文件位置。
```

- [TracingConfiguration](#apiserver-k8s-io-v1alpha1-TracingConfiguration)

- [TracingConfiguration](#apiserver-k8s-io-v1beta1-TracingConfiguration)

<p>
<!--
TracingConfiguration provides versioned configuration for OpenTelemetry tracing clients.
-->
TracingConfiguration 为 OpenTelemetry 跟踪客户端提供了不同版本的配置。
</p>

<table class="table">
<thead><tr><th width="30%">Field</th><th>Description</th></tr></thead>
<tbody>
 
<tr><td><code>endpoint</code><br/>
<code>string</code>
</td>
<td>
   <p>
   <!--
   Endpoint of the collector this component will report traces to.
   The connection is insecure, and does not currently support TLS.
   Recommended is unset, and endpoint is the otlp grpc default, localhost:4317.
   -->
   采集器的端点，此组件将向其报告跟踪信息。
   连接不安全，目前不支持 TLS。
   推荐不设置，端点为 otlp grpc 默认值 localhost:4317。
   </p>
</td>
</tr>
<tr><td><code>samplingRatePerMillion</code><br/>
<code>int32</code>
</td>
<td>
   <p>
   <!--
   SamplingRatePerMillion is the number of samples to collect per million spans.
   Recommended is unset. If unset, sampler respects its parent span's sampling
   rate, but otherwise never samples.
   -->

>    SamplingRatePerMillion 是每百万 span 中采集的样本数。

Reviewer 2: ```suggestion
   samplingRatePerMillion 是每百万 span 中采集的样本数。
```

---
title: kube-apiserver 配置 (v1beta1)
content_type: tool-reference
package: apiserver.k8s.io/v1beta1
auto_generated: true
---
<!-- 
title: kube-apiserver Configuration (v1beta1)
content_type: tool-reference
package: apiserver.k8s.io/v1beta1
auto_generated: true
-->

<!-- 
<p>Package v1beta1 is the v1beta1 version of the API.</p> 
-->
<p>包 v1beta1 包含 API 的 v1beta1 版本。</p>

<!-- 
## Resource Types  
-->
## 资源类型   {#resource-types}

- [EgressSelectorConfiguration](#apiserver-k8s-io-v1beta1-EgressSelectorConfiguration)
- [TracingConfiguration](#apiserver-k8s-io-v1beta1-TracingConfiguration)

>   

Reviewer 3: Just change it to one blank line, and the same goes for other places
